pywbem/_listener.py

```diff
@@ -702,7 +704,7 @@ def http_port(self):
     @property
     def https_port(self):
         """
-        The HTTPS port this listener can be reached at, as an :term:`integer`.
+        :term:`integer`: The HTTPS port this listener can be reached at.
```

Comment:
You gotta love english and split infinitives.  Change to The HTTPS port at which this listener can be reached.

Reply:
Changed as proposed, in all 6 occurrences.